packages/console/src/components/MultilineInput/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import * as textButtonStyles from '@/components/TextButton/index.module.scss';
+import Minus from '@/icons/Minus';
+
+import IconButton from '../IconButton';
+import TextInput from '../TextInput';
+import * as styles from './index.module.scss';
+
+type Props = {
+  value: string[];
+  onChange: (value: string[]) => void;
+};
+
+const MultilineInput = ({ value, onChange }: Props) => {
+  const { t } = useTranslation(undefined, {
+    keyPrefix: 'general',
+  });
+
+  const fields = useMemo(() => {
+    if (value.length === 0) {
+      return [''];
+    }
+
+    return value;
+  }, [value]);
```

Comment:
no need for this?

Reply:
per offline discussion, key this to handle the corner case of `value.length === 0`.